samples/16.customization-selectable-activity/README.md

```diff
@@ -0,0 +1,3 @@
+# Selectable Chat Activity
+
+In this sample, we show how to add an activity middleware to allow the user to select an activity.
```

Comment:
Can we add more detail in the `README.md`? Including `npm i`, what the user can try out when testing this sample, any steps you think are pertinent. :)

Reply:
Will do!